rpi_backlight.py

```diff
@@ -60,9 +60,9 @@ def get_power() -> bool:
     """Return wether the display is powered on or not."""
     # 0 is on, 1 is off
     if int(_get_value("tinker_mcu_bl")) == 0:
-        return "Off"
+        return 1
     else:
-        return "On"
+        return 0
```

Comment:
Still not a `bool`! You'd want this:

```python
    if int(_get_value("tinker_mcu_bl")) == 0:
        return True
    else:
        return False
```

Or shorter:

```python
    return int(_get_value("tinker_mcu_bl")) == 0
```

But of course `_get_value("tinker_mcu_bl")` must be changed.

Reply:
Sorry what do you mean by

> But of course _get_value("tinker_mcu_bl") must be changed.

>

Reply:
As I wrote in my comment here https://github.com/linusg/rpi-backlight/pull/13#pullrequestreview-223543099, we need some more functionality - just renaming the base path and all files would break compatibility with any Raspberry Pi!
(since `/sys/devices/platform/ff150000.i2c/i2c-3/3-0045/tinker_mcu_bl` doesn't exist, only `/sys/class/backlight/rpi_backlight/bl_power`)

Reply:
I've put in an initial switch statement. Haven't tested it yet. Can you sanity check it first before I do a polish pass?